app_dart/test/request_handlers/push_build_status_to_github_test.dart

```diff
@@ -134,28 +181,49 @@ void main() {
         });
 
         test('if status has not changed since last update', () async {
-          final PullRequest pr = newPullRequest(id: 123, sha: 'abc');
-          prsFromGitHub = <PullRequest>[pr];
+          githubPullRequestsMaster = <int>[1];
+          final PullRequest pr = newPullRequest(id: 1, sha: '1');
+          githubBranches = <String>['master'];
           buildStatusProvider.cumulativeStatus = BuildStatus.succeeded;
           final GithubBuildStatusUpdate status =
               newStatusUpdate(pr, BuildStatus.succeeded);
           db.values[status.key] = status;
+          branchHttpClient.request.response.body = branchRegExp;
+          final Body body = await tester.get<Body>(handler);
+          expect(body, same(Body.empty));
+          expect(status.updates, 0);
+          expect(log.records.where(hasLevel(LogLevel.WARNING)), isEmpty);
+          expect(log.records.where(hasLevel(LogLevel.ERROR)), isEmpty);
+        });
+
+        test('if there is no pr found for a targeted branch', () async {
+          githubPullRequestsMaster = <int>[1];
+          final PullRequest pr = newPullRequest(id: 1, sha: '1');
+          githubBranches = <String>['flutter-0.0-candidate.0'];
+          buildStatusProvider.cumulativeStatus = BuildStatus.succeeded;
+          final GithubBuildStatusUpdate status =
+              newStatusUpdate(pr, BuildStatus.failed);
+          db.values[status.key] = status;
+          branchHttpClient.request.response.body = branchRegExp;
           final Body body = await tester.get<Body>(handler);
           expect(body, same(Body.empty));
           expect(status.updates, 0);
+          expect(status.status, BuildStatus.failed.githubStatus);
           expect(log.records.where(hasLevel(LogLevel.WARNING)), isEmpty);
           expect(log.records.where(hasLevel(LogLevel.ERROR)), isEmpty);
         });
       });
 
```

Comment:
There should be a test where there are multiple branches (e.g. master and dev) to make sure both get updates pushed to them

Reply:
Added.